featuretools/tests/primitive_tests/natural_language_primitives_tests/test_natural_language_primitives_terminate.py

```diff
@@ -0,0 +1,64 @@
+import signal
+from platform import system
+
+import pandas as pd
+import pytest
+
+from featuretools.primitives import (
+    CountString,
+    MeanCharactersPerWord,
+    MedianWordLength,
+    NumberOfHashtags,
+    NumberOfMentions,
+    NumberOfUniqueWords,
+    NumberOfWordsInQuotes,
+    NumUniqueSeparators,
+    PunctuationCount,
+    TitleWordCount,
+    TotalWordLength,
+    UpperCaseCount,
+    WhitespaceCount,
+)
+
+TIMEOUT_THRESHOLD = 20
+
+if system == "Linux" or system == "Darwin":
```

Comment:
Check out pytest's skipif: https://docs.pytest.org/en/6.2.x/reference.html?highlight=pytest%20skip#pytest-mark-skipif-ref

Also I think given these codecov warnings, `os.sytems` isn't "Linux" for our standard CI runs